app/src/main/java/com/android/wardrobeManager/clothing_classes/Shirt.java

```diff
@@ -0,0 +1,49 @@
+package com.android.wardrobeManager.clothing_classes;
+
+import android.os.Parcel;
+
+import com.android.wardrobeManager.ClothingItem;
+
+public class Shirt extends ClothingItem {
+    public Shirt(int color) {
+        super(color);
+    }
+
+    public static final Creator<Shirt> CREATOR = new Creator<Shirt>() {
+        @Override
+        public Shirt createFromParcel(Parcel in) {
+            return new Shirt(in);
+        }
+
+        @Override
+        public Shirt[] newArray(int size) {
+            return new Shirt[size];
+        }
```

Comment:
You have to initialize the slots in this array to shirt object. Just use default constructor if there is one. Add one if there is not.